osdep/terminal-win.c

```diff
@@ -204,6 +204,9 @@ void terminal_setup_getch(struct input_ctx *ictx)
     }
 }
 
+static bstr write_console_buf;
+static wchar_t *write_console_wbuf;
```

Comment:
needs to be thread-local, otherwise unsafe

Reply:
Our whole terminal printf and siblings is synchronized. Well, except kitty and tct. You are right it should be thread_local, I need to think how to free it in this case...

Reply:
`pthread_key_create` probably

Reply:
It is windows, we don't have things like that implemented :)

Reply:
Ok, added it using FLS api.

I could extend mp_thread abstraction, but I don't feel like it for this small one time occasion. If there are more use-cases for TLS we can think of proper multi platform implementation.